wye/workshops/templatetags/workshop_action_button.py

```diff
@@ -29,16 +41,45 @@ def show_accepted_button(workshop, use):
 register.filter(show_accepted_button)
 
 
-def show_feedback_button(workshop, use):
+def show_feedback_button(workshop, user):
+
+    is_requester = False
+    is_presenter = False
+    is_completed = False
+
     if workshop.status == WorkshopStatus.COMPLETED:
-        return True
+
+        requester_members = workshop.requester.user.values()
+        presenters = workshop.presenter.values()
+
+        for member in requester_members:
+            if user.username == member['username']:
+                is_requester = True
+
+        for presenter in presenters:
```

Comment:
We can replace for loop with if condition 
if user in workshop.presenter.all():